PageObjectPattern/src/test/java/api/stepdefinitions/PostPlayListStepDefinitions.java

```diff
@@ -0,0 +1,50 @@
+package api.stepdefinitions;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThat;
+
+import org.junit.Assert;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.Assert.*;
+
+import apiSerenityStep.AppAuthStep;
+import apiSerenityStep.BaseStep;
+import apiSerenityStep.SpotifyApiStep;
+import io.cucumber.java.en.*;
+import io.restassured.response.ValidatableResponse;
+import net.thucydides.core.annotations.Steps;
+
+public class PostPlayListStepDefinitions {
+
+    @Steps
+    AppAuthStep auth;
+
+    @Steps
+    SpotifyApiStep api;
+
+    @Given("User has Auth token")
+    public void getToken() {
+        System.out.println("Step1: Getting token");
+        auth.getAuthorized();
+    }
+
+    @When("post request is sent to {word} with {}")
```

Comment:
Better to update description to be more specific since this method is only about create playlist.